client/src/lib/firebase/app.ts

```diff
@@ -37,6 +38,9 @@ const app = initializeApp(firebaseConfig);
 export const db = getFirestore(app);
 export const auth = getAuth(app);
 
+auth.onAuthStateChanged((user) => {
+  authStore.set(user);
+});
```

Comment:
will this code be run every time this component is loaded?
would it be better to put it in `$stores/auth`?

Reply:
this was me being dumb because I thought this was +page.svelte for some reason